src/gmail.js

```diff
@@ -3984,6 +4036,19 @@ var Gmail = function(localJQuery) {
                 show_cc: "span.aB.gQ.pE",
                 show_bcc: "span.aB.gQ.pB"
             };
+
+            // Post "peoplekit"
+            if (this.$el.find("textarea[name=to]").length === 0 && ["to", "cc", "bcc"].includes(lookup)) {
+                const configPeoplekit = {
+                    to:"div[name=to] input",
```

Comment:
Not having seen the UI for the "peoplekit" update... Are these `input`-element hidden? Can the selector be used for updating values or just extracting them?